conda_smithy/configure_feedstock.py

```diff
@@ -919,11 +919,6 @@ def render_circle(jinja_env, forge_config, forge_dir, return_metadata=False):
             os.path.join(forge_dir, ".circleci", "checkout_merge_commit.sh"),
             os.path.join(forge_dir, ".circleci", "fast_finish_ci_pr_build.sh"),
         ],
-        "linux": [
-            os.path.join(forge_dir, ".scripts", "run_docker_build.sh"),
-            os.path.join(forge_dir, ".scripts", "build_steps.sh"),
-        ],
-        "osx": [os.path.join(forge_dir, ".circleci", "run_osx_build.sh")],
```

Comment:
smithy now handles the deletion of these files at the start and then they are rendered on each provider in the `_render_template_exe_files`. thus I am removing them here and for other ci providers below.